docs/sources/developers/plugins/migration-guide.md

```diff
@@ -15,6 +15,8 @@ This guide helps you identify the steps you need to take based on the Grafana ve
 - [From version 8.3.x to 8.4.x](#from-version-83x-to-84x)
   - [Value Mapping Editor has been removed from @grafana-ui library](#value-mapping-editor-has-been-removed-from-grafana-ui-library)
   - [Thresholds Editor has been removed from @grafana-ui library](#thresholds-editor-has-been-removed-from-grafana-ui-library)
+  - [8.3 Deprecations](#83-deprecations)
```

Comment:
Should this be 8.4 as milestone and backport label?

Reply:
☝️

Reply:
Yes! Thanks for highlighting this 🙏🏻